Current master has a failing test

```
esc@omnitech ~/git/python-blosc (git)-[master] % nosetests --with-doctest blosc
...............F...........S...
======================================================================
FAIL: test_bitshuffle_not_multiple (blosc.test.TestCodec)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/esc/git/python-blosc/blosc/test.py", line 320, in test_bitshuffle_not_multiple
    self.assertEqual(last_xx, b'\x01\x01\x01')
AssertionError: b'\x01\x00\x00' != b'\x01\x01\x01'

----------------------------------------------------------------------
Ran 31 tests in 0.420s

FAILED (SKIP=1, failures=1)
```

This is the test for the infamous #133 - however I don't understand what the test should be testing.

Interestingly enough, this only happens when ALSO running the doctests:

```
1 esc@omnitech ~/git/python-blosc (git)-[master] % nosetests                                                                                                                                                             :(
...............S...
----------------------------------------------------------------------
Ran 19 tests in 0.150s

OK (SKIP=1)
esc@omnitech ~/git/python-blosc (git)-[master] % nosetests --with-doctest 
...............F...........S...
======================================================================
FAIL: test_bitshuffle_not_multiple (blosc.test.TestCodec)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/esc/git/python-blosc/blosc/test.py", line 320, in test_bitshuffle_not_multiple
    self.assertEqual(last_xx, b'\x01\x01\x01')
AssertionError: b'\x01\x00\x00' != b'\x01\x01\x01'

----------------------------------------------------------------------
Ran 31 tests in 0.592s

FAILED (SKIP=1, failures=1)
```

--- a/blosc/toplevel.py
+++ b/blosc/toplevel.py
@@ -111,6 +111,7 @@ def set_blocksize(blocksize):
     --------
 
     >>> blosc.set_blocksize(512)
+    >>> blosc.set_blocksize(0)
 
     """
 

